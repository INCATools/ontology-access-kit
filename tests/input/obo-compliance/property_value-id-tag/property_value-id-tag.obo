format-version: 1.4
ontology: property_value-id-tag

[Term]
id: X:1
property_value: id "X:1" xsd:string

[Typedef]
id: R:1
xref: http://www.geneontology.org/formats/oboInOwl#id
is_metadata_tag: true

