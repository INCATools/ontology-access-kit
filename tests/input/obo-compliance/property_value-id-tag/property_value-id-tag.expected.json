{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/property_value-id-tag.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://purl.obolibrary.org/obo/R_1",
      "type" : "PROPERTY",
      "meta" : {
        "xrefs" : [ {
          "val" : "http://www.geneontology.org/formats/oboInOwl#id"
        } ],
        "basicPropertyValues" : [ {
          "pred" : "http://www.geneontology.org/formats/oboInOwl#is_metadata_tag",
          "val" : "true"
        } ]
      }
    }, {
      "id" : "http://purl.obolibrary.org/obo/X_1",
      "type" : "CLASS",
      "meta" : {
        "basicPropertyValues" : [ {
          "pred" : "http://purl.obolibrary.org/obo/property_value-id-tag#id",
          "val" : "X:1"
        } ]
      }
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasDbXref",
      "lbl" : "database_cross_reference",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#id",
      "lbl" : "id",
      "type" : "PROPERTY"
    } ]
  } ]
}